docs/about/introduction.mdx

```diff
@@ -21,11 +22,11 @@ Meshtastic® is a project that enables you to use inexpensive LoRa radios as a l
 
 ## How it works
 
-The underlying technology, LoRa, is a long range radio protocol available to most regions without requiring certification like HAM operators.
+The underlying technology, LoRa, is a long range radio protocol available to most regions without requiring additional licensing or certification, like HAM operators.
 
-The radios automatically rebroadcast messages they receive in order to create a mesh network so that everyone in the group can receive messages - even from the furthest member. Depending on settings used, the Meshtastic mesh can sustain up to 80 device nodes.
+The radios automatically link up and rebroadcast messages they receive in order to create a mesh network so that everyone in the group can receive messages - even from the furthest member. Depending on settings used, the Meshtastic mesh can sustain up to 80 device nodes.
 
-Radios can be paired to a single phone so that your friends and family are able to address a message to your specific radio. Each device supports a connection from a single user at a time.
+Meshtastic radios are paired to a single phone so that your friends and family are able to address a message to your specific radio. Each device supports a connection from a single user at a time.
```

Comment:
I think this should be referred, as phones are only one option to interact with the mesh

Reply:
accepted